game_api.txt

```diff
@@ -228,6 +230,9 @@ The doors mod allows modders to register custom doors and trapdoors.
 
 	description = "Trapdoor description",
 	inventory_image = "mod_trapdoor_inv.png",
+	closed = {} -- Nodebox for closed model
+	opened = {} -- Nodebox for opened model
+	-- (see https://github.com/minetest/minetest_game/pull/2371 for furthur info)
```

Comment:
Ew, this looks awful.
It shouldn't be needed to open a link to understand something. The documentation should explain everything.

Reply:
OK, I'll modify it.

Reply:
I agree, don't add links, all essential info should be in the comment.